app/src/main/java/com/ncnf/main/MainActivity.java

```diff
@@ -17,37 +20,76 @@
 
     private BottomNavigationView navigationBar;
 
+    final FragmentManager fragmentManager = getSupportFragmentManager();
+    private Fragment homeFragment;
+    private Fragment mapFragment;
+    private Fragment feedFragment;
+
+    private Fragment activeFragment;
+
```

Comment:
Shouldn't the home/map/feed fragment be final since there are not going to change ?

Reply:
I tried, but the problem is that I setup them in the onCreate function and there is no constructor for this. This is the only way it was accepted by android studio, without having to recreate the activity, thus loosing the loaded fragments.

Reply:
Ok that makes sense. We will keep like this.

Reply:
WHAT A PLAYYYEEEER